src/HttpCache.php

```diff
@@ -173,12 +173,12 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
 
     /**
      * Validates if the HTTP cache has valid content.
-     * If both Last-Modified and ETag are `null`, it returns `false`.
+     * If both `Last-Modified` and `ETag` are `null`, it returns `false`.
```

Comment:
Markdown inside code comments is not rendered.

Reply:
It depends on what we are rendering with. In PHPStorm that's currently true but [a number of issues are opened about it](https://youtrack.jetbrains.com/issues?q=project:%20WI%20state:%20Open%20%7Bmarkdown%20phpdoc%7D) so eventually it will be fixed. In Yii apidoc it's supported.

Reply:
I commented on the wrong line. See https://github.com/yiisoft/yii-middleware/pull/88/files#r1199987083 instead.